src/_guides/language/effective-dart/style.md

```diff
@@ -362,34 +356,34 @@ Doing so avoids the [dangling else][] problem.
 
 [dangling else]: http://en.wikipedia.org/wiki/Dangling_else
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/style_good.dart (curly-braces)"?>
 {% prettify dart %}
-if (true) {
-  print('sanity');
+if (isWeekDay) {
+  print('Bike to work!');
 } else {
-  print('opposite day!');
+  print('Go dancing or read a book!');
 }
 {% endprettify %}
-</div>
 
 There is one exception to this: `if` statements with no `else` clause that fit
 on one line may omit the braces.
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/style_good.dart (one-line-if)"?>
 {% prettify dart %}
 if (arg == null) return defaultValue;
 {% endprettify %}
-</div>
 
 These are typically used for "guard" code that returns or breaks if the
 condition is met. But they're also fine for expressions, as long as the entire
 `if` statement and the expression fit on one line.
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/style_good.dart (one-line-if-expr)"?>
 {% prettify dart %}
-if (parameter == null) parameter = defaultValue;
+if (parameter > limit) parameter = defaultValue;
```

Comment:
`if (foo == null) foo = expr` would more succinctly be written as `foo ??= expr`, so I changed the example a bit (since we're meant to be illustrating single-line `if` statements).